dev-env/run-docker-compose.sh

```diff
@@ -49,6 +97,13 @@ if [[ "${1:-""}" == 'restart' ]]; then
   exit
 fi
 
+if [[ "${1:-""}" == 'stop' ]]; then
+  echo "Stopping dev containers..."
+  compose_down
```

Comment:
This suggestion also implies to replace all occurrences of  `compose_down` in the script.

```suggestion
  compose_stop
```